blocks/tabs/tabs.css

```diff
@@ -0,0 +1,297 @@
+.block.tabs .tabs-list {
+    display: flex;
+    max-width: 100%;
+    overflow-x: auto;
+    background-color: var(--primary);
+    justify-content: left;
+    padding: var(--space-8px) 0;
+}
+  
+.block.tabs .tabs-list button {
+    flex: 0 0 auto;
+    border: var(--space-1px) solid var(--dark-color);
+    padding: var(--space-8px) 1rem;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    color: var(--color-white);
+    font-family: var(--body-font-family);
+}
+  
+.block.tabs .tabs-list button[aria-selected="true"] {
```

Comment:
is it possible to do this adding a class on button ?

Reply:
it is from OOTB Tabs block , so didn't feel to change it.